src/transformers/models/mamba/modeling_mamba.py

```diff
@@ -715,3 +741,156 @@ def forward(
             cache_params=mamba_outputs.cache_params,
             hidden_states=mamba_outputs.hidden_states,
         )
+
+
+class MambaClassificationHead(nn.Module):
+    """Head for sentence-level classification tasks."""
+
+    def __init__(self, config):
+        """Initialize the head."""
+        super().__init__()
+        self.activation = ACT2FN[config.hidden_act]
+        self.dense = nn.Linear(config.hidden_size, config.hidden_size)
+        self.dropout = nn.Dropout(config.classifier_dropout)
+        self.out_proj = nn.Linear(config.hidden_size, config.num_labels)
+
+        self.config = config
+
+    def forward(self, features, **kwargs):
+        """Forward pass."""
+        # Pooling is done by the forward pass in `MambaForSequenceClassification`
+        x = features
+        x = self.dropout(x)
+        x = self.dense(x)
+        x = self.activation(x)
+        x = self.dropout(x)
+
+        return self.out_proj(x)
```

Comment:
let's avoid single letter variables please

Reply:
Got it, fixed.